src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+}
```

Comment:
1. please use classes to style elements
2. you don't need to style img, it has a proper styles, you can just style`<a>` element and use class `header__logo-link` or just `header__logo` to do it